### DISCUSSION TITLE
node-red command not found

### DISCUSSION THREAD

**Discussion by micma42** [ct-Open-Source/ct-Smart-Home#54] [Q&A]
Hi,
I am trying to add password protection to the Node-Red-Backend as described in [c't FAQ Node-Red](https://www.heise.de/select/ct/2019/19/1567945797866843). According to the [Node-Red Doku](https://nodered.org/docs/user-guide/node-red-admin), node-red-admin does not have to be installed separately, but is included in the node-red command since version 1.1.0. My recently cloned ct-Smart-Home has version 1.2.9.

I cannot get these commands to run, though. This is what I did:
`pi@raspberrypi:~ $ docker exec -it ct-smart-home_nodered_1 /bin/bash`
`bash-5.0# node-red admin hash-pw`
`bash: node-red: command not found`

I also tried to install node-red-admin with npm in the docker bash. This did execute with lots of warnings and errors, but then also yielded a 
`bash-5.0# node-red-admin hash-pw`
`bash: node-red-admin: command not found`

Hints on what I missed are highly appreciated.

**Comment by besbar** [comment]
https://www.npmjs.com/package/node-red-admin
